handler/handler.go

```diff
@@ -71,6 +71,17 @@ func New(cfg config.Proxy, bp httputil.BufferPool, prov service.Authorizationd)
 
 			*r = *req
 		},
+		ModifyResponse: func(res *http.Response) error {
+			if cfg.OriginLog.StatusCode.Enable {
+				for _, statusCode := range cfg.OriginLog.StatusCode.Exclude {
+					if statusCode == res.StatusCode {
+						return nil
+					}
+				}
+				glg.Infof("Origin request: %s %s, Response: status code: %d", res.Request.Method, res.Request.URL, res.StatusCode)
+			}
+			return nil
+		},
```

Comment:
```suggestion
		ModifyResponse: modifyResponse,
```

maybe better to define the function before `&httputil.ReverseProxy{}`?
```go
modifyResponse := func(res *http.Response) error {
    ...
}
if !cfg.OriginLog.StatusCode.Enable {
    modifyResponse = nil
}
```